test_opensearchpy/test_server/test_plugins/test_notification.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+
+from __future__ import unicode_literals
+
+from .. import OpenSearchTestCase
+
+
+class TestNotificationPlugin(OpenSearchTestCase):
+    async def test_create_channel_notification(self) -> None:
```

Comment:
- @AbitraryYu, please remove async from all the test cases in this file.
- After removing async, you might notice that some test cases fail. Please fix them.
- For example, test_delete_channel_configuration may fail. Remember, each test case is isolated, so create the configuration before deleting it within the same test case. See [this reference](https://github.com/opensearch-project/opensearch-py/blob/main/test_opensearchpy/test_server/test_plugins/test_index_management.py#L108). Thank you.